Docs/source/running_cpp/pwfa.rst

```diff
@@ -123,6 +123,12 @@ Plotfiles and snapshots
     The diagnostics results are analysed and stored in snapshots at each time step and so it is best to make sure that the run does not end before filling the final snapshot.
 
 
+    .. note::
+
+       You may find example python scripts (in `Tools <>_`) that do the analysis for convergence scans of the WarpX multi stage plasma wakefield acceleration configuration simulations.
```

Comment:
Note: link target in `Tools <....>_` is probably still to do :)

Reply:
Yes, sorry for that.
Will fix it asap 😄 thanks for catching it @ax3l !